cookbook_se/recipes/01_basic/folders.py

```diff
@@ -0,0 +1,80 @@
+"""
+11: Work with folders
+---------------------
+
+Please also see the entry on files. After files, folders are the next fundamental grouping users might find themselves working with. Flyte's IDL supports folders as what we call a multi
```

Comment:
a multi...?

Reply:
yeah man don't you know what a multi is?